spec/praxis/plugins/praxis_mapper_plugin_spec.rb

```diff
@@ -0,0 +1,81 @@
+require 'spec_helper'
+
+describe Praxis::Plugins::PraxisMapperPlugin do
+
+  subject(:plugin) { Praxis::Plugins::PraxisMapperPlugin::Plugin.instance }
+  let(:config) { plugin.config }
+
+  context 'Plugin' do
+    context 'configuration' do
+      subject { config }
+      its(:log_stats) { should eq 'skip' }
+
+      its(:repositories) { should have_key("default") }
+
+      context 'default repository' do
+        subject(:default) { config.repositories['default'] }
+        its(['type']) { should eq 'sequel' }
+        its(['connection_settings']) do
+          should eq('adapter' => 'sqlite','database' => ':memory:')
+        end
+      end
+
+    end
+  end
+
+  context 'functional test' do
+
+    def app
+      Praxis::Application.instance
+    end
+
+    let(:session) { double("session", valid?: true)}
+
+    around(:each) do |example|
+      orig_level = Praxis::Application.instance.logger.level
+      Praxis::Application.instance.logger.level = 2
+      config.log_stats = 'detailed'; plugin.setup!
+      example.run
+      config.log_stats = 'skip'; plugin.setup!
+      Praxis::Application.instance.logger.level = orig_level
+    end
```

Comment:
This is needed in order to override what was originally configured at boot time for the plugin (log_stats=skip), so we can test logging.